webpack/app/components/TableCells.coffee

```diff
@@ -76,6 +75,7 @@ class TableCells extends React.Component
           value={uid}
           disabled={@props.disabled}
           checked={@props.selected}
+          tabIndex={"-1"}
```

Comment:
here again. Just -1 should be fine. The curly brackets are there to evaluate JS code inline